products/ansible-core.md

```diff
@@ -0,0 +1,45 @@
+---
+permalink: /ansible-core
+layout: post
+title: Ansible-core
+command: ansible --version
+releasePolicyLink: https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html
+changelogTemplate: https://github.com/ansible/ansible/blob/stable-__RELEASE_CYCLE__/changelogs/CHANGELOG-v__RELEASE_CYCLE__.rst
+releaseDateColumn: true
+sortReleasesBy: "releaseCycle"
+activeSupportColumn: false
+eolColumn: Supported
+iconSlug: ansible
+auto:
+  git: https://github.com/ansible/ansible.git
+releases:
+  - releaseCycle: "2.13"
+    release: 2021-11-05
+    eol: false
+    latest: "v2.13.0b0"
+  - releaseCycle: "2.12"
+    release: 2021-11-05
+    eol: false
+    latest: "2.12.4"
+  - releaseCycle: "2.11"
+    release: 2021-04-26
+    eol: true
+    latest: "2.11.10"
+  - releaseCycle: "2.10"
+    release: 2020-08-13
+    eol: true
+    latest: "2.10.17"
+  - releaseCycle: "2.9"
+    release: 2019-10-31
+    eol: false
+    latest: "2.9.27"
+
+---
+
+> [Ansible](https://ansible.com) is an open-source software provisioning, configuration management, and application-deployment tool enabling infrastructure as code. It runs on many Unix-like systems, and can configure both Unix-like systems as well as Microsoft Windows.
+
+The `ansible` package (which this page tracks) typically gets 2 major releases every year. A new minor version is released every 3 weeks. Maintenance fixes are guaranteed for only the latest major release. `ansible-core`).
+
+See the [Ansible Roadmap][roadmap] for upcoming release details.
+
+[roadmap]: https://docs.ansible.com/ansible-core/devel/roadmap/ansible_core_roadmap_index.html
```

Comment:
```suggestion
[roadmap]: https://docs.ansible.com/ansible-core/devel/roadmap/ansible_core_roadmap_index.html

## Python Compatibility

| ansible-core Version | Minimum Python Version (controller) | Minimum Python Version (modules) |
|----------------------|-------------------------------------|----------------------------------|
| 2.9                  | 2.7 or 3.5                          | 2.6 or 3.5                       |
```

@gundalow Something like this? If it's OK, please accept / merge it and I'll add the other `ansible-core` versions and would add the same to `ansible`.

Reply:
Looks great, thanks.
Maybe have as `2.6+ or 3.5+`?
I know you have minimum in the title, though this might be clearer?

Reply:
> Maybe have as `2.6+ or 3.5+`?
> I know you have minimum in the title, though this might be clearer?

https://endoflife.date/tomcat doesn't, either. That's the reason why I didn't use `2.6+ or 3.5+`. I've used this as a template because you've mentioned it. It wouldn't hurt, but it would be a little bit redundant.

What about removing the `Minimum` and use `2.6+ or 3.5+`? I think this would be even better. This would also allow us to do something like `2.6+ or 3.5 - 3.whatever` if there's a problem with Python 3.whatever+1 without changing the header.

Reply:
Ah, I guess we should be consistent with Tom at. Good spot.

Reply:
So keep it like this? Then I'd add the Python versions for other `ansible-core` versions. In a second step, I'd do the same for `ansible`.

Reply:
Yes, what you have is great, please continue.
Thanks for your help with this.

Reply:
I've updated my suggested change for versions 2.10 to 2.12. I hope I've found the correct requirements.